config/webpack/prod.js

```diff
@@ -90,7 +52,7 @@ const config = {
     plugins: [
         //new webpack.HotModuleReplacementPlugin(),
         new webpack.DefinePlugin({
-            'process.env.NODE_ENV': '"production"',
+            'process.env.NODE_ENV': 'production',
```

Comment:
This breaks `yarn build/production`, should be as it was before (`'"production"'`).

Reply:
Thats nice, ci passed and my build local worked. 
But anyway i’ll put it back, thought its a typo